src/set_theory/ordinal_notation.lean

```diff
@@ -5,10 +5,117 @@ Author: Mario Carneiro
 
 Ordinal notations (constructive ordinal arithmetic for ordinals < ε₀).
 -/
-import set_theory.ordinal
+import set_theory.ordinal_arithmetic
 open ordinal
```

Comment:
```suggestion
/-!
# Ordinal notations (constructive ordinal arithmetic for ordinals < ε₀)

This file contains definitions of Cantor Normal Form (`ordinal.CNF`) and ordinal notations (`onote`).

-/
open ordinal
```

Actually, should the CNF stuff go in this file? It doesn't seem to be used at all here @digama0

Reply:
I think `CNF` should go along with the ordinal logarithm, which I guess is `set_theory.ordinal_arithmetic`. `ordinal_notation` is specifically for constructive variants on the ordinal functions (although its core representation format is closely related to a recursive application of `CNF`).

Reply:
To me the CNF is also a form of notation on ordinals, that's why I moved it to this file. But since you both agree, I'll move it back to `ordinal_arithmetic`.

Reply:
Done.

Reply:
Yeah, I didn't know this before looking at this file yesterday, but the issue is that an ["ordinal notation"](https://en.wikipedia.org/wiki/Ordinal_notation) is not simply mathematical notation on ordinals.